src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -252,6 +253,7 @@ export const NewAccountTosScene = (props: NewAccountTosProps) => {
           return
         }
         onLogin(account)
+        dispatch(completeLogin(account))
```

Comment:
Same here - we don't need to call `onLogin` because `completeLogin` does that too.